out_writeapi.go

```diff
@@ -160,6 +164,20 @@ func checkResponses(curr_ctx context.Context, currQueuePointer *[]*managedwriter
 	return 0
 }
 
+type MWManagedStream interface {
+	AppendRows(ctx context.Context, data [][]byte, opts ...managedwriter.AppendOption) (*managedwriter.AppendResult, error)
```

Comment:
Even though not needed at the moment, could you define this interface to include the other public methods: FlushRows, Finalize, and Close? That way these will be ready when needed in future. You can have the default implementation simply call the real methods, and the mock one simply return nil.

Reply:
Done.